packages/twenty-server/src/engine/core-modules/twenty-config/config-variables.ts

```diff
@@ -346,6 +348,7 @@ export class ConfigVariables {
     description: 'Type of storage to use (local or S3)',
     type: ConfigVariableType.ENUM,
     options: Object.values(StorageDriverType),
+    isEnvOnly: true,
```

Comment:
This is the kind of variable that we built the admin panel for, would be a shame if they really don't work